### PR TITLE
Fix cluster autoscaler version

### PR DESCRIPTION
**Pull request by betabandido**

### CODE REVIEW

File: cluster-autoscaler-version.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  exit 1
+fi
+
+VERSION=$(git \
+    -c 'versionsort.suffix=-' \
+    ls-remote \
+    --exit-code \
+    --tags \
+    --sort='v:refname' \
+    git@github.com:kubernetes/autoscaler.git \
```
Comment (xorxsan) [inline cluster-autoscaler-version.sh:15]: Would it be possible to use `https://github.com/kubernetes/autoscaler.git` instead?

File: cluster-autoscaler.tf [hunk 2]
```diff
@@ -1,12 +1,22 @@
+data "external" "cluster_autoscaler_version" {
```
Comment (xorxsan) [inline cluster-autoscaler.tf:1]: Shouldn't this be a variable instead?

Comment (betabandido) [reply cluster-autoscaler.tf:1]: That wouldn't work. You cannot execute scripts using a variable.

Comment (xorxsan) [reply cluster-autoscaler.tf:1]: Sorry, I meant using a variable instead of executing a script.

Comment (betabandido) [reply cluster-autoscaler.tf:1]: But, then you'll need to go to the github repo, find the corresponding matching version for the cluster version that has been used, and finally use that value to set the variable. That sounds like quite a bit of work.

**Pull request merged**